cpp/include/raft/core/mdarray.hpp

```diff
@@ -109,7 +110,8 @@ using padded_layout = detail::stdex::layout_padded_general<
   order>;
 
 template <typename ElementType, storage_order_type order>
-using padded_matrix = mdspan<ElementType, matrix_extent, padded_layout<ElementType, order>>;
+using padded_matrix =
```

Comment:
Given that `uint32_t` is the default, would you consider just relying on the default here, in order to avoid confusion and possible errors?

Reply:
@mhoemmen , thank you very much for your input. I have tried to apply all of your suggestions and also modified the matrix `LinewiseOp `operations to utilize the `aligned_mdspan`.